README.md

```diff
@@ -1 +1,47 @@
 # Conduit Connector Azure Storage
+
+## General
+
+The Azure Storage plugin is one of [Conduit](https://github.com/ConduitIO/conduit) plugins.
+It currently provides only source connector.
+
+## How to build it
+
+Run `make`.
+
+## Source
```

Comment:
It would be good to add more detailed info about how snapshot and CDC work. E.g. what happens when a snapshot is stopped, how does CDC detect changes, does it always pull all objects etc.

Reply:
Updated with additional information: 856eea2